src/DriverFactory.php

```diff
@@ -41,9 +36,8 @@ protected function createWithConfig(ContainerInterface $container, $configKey)
             $config['paths'] = [$config['paths']];
         }
 
-        if ($config['class'] === AnnotationDriver::class || is_subclass_of($config['class'], AnnotationDriver::class)) {
-            $this->registerAnnotationLoader();
```

Comment:
Outstanding concern - original discussion:

https://github.com/Roave/psr-container-doctrine/pull/3#discussion_r376152219

Do we need to keep this @Ocramius ?

Reply:
I'd keep autoloader registration: it's a very basic concept, and we can actually enable it by default at all times (not even conditionally, tbh)

Reply:
OK I'll add it back in :+1: